### PR TITLE
test: refactor nav spec to use sidebar pagemodels [INFENG-683]

### PR DESCRIPTION
**Pull request by JComins000**
## Ticket

[INFENG-683]

## Description

refactor to use sidebar pagemodels

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[INFENG-683]: https://hpe-aiatscale.atlassian.net/browse/INFENG-683?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/e2e/tests/navigation.spec.ts [hunk 1]
```diff
@@ -7,13 +7,14 @@ import { SignIn } from 'e2e/models/pages/SignIn';
 import { Workspaces } from 'e2e/models/pages/Workspaces';
 
 test.describe('Navigation', () => {
-  const USERNAME = process.env.PW_USER_NAME ?? '';
```
Comment (JComins000) [inline webui/react/src/e2e/tests/navigation.spec.ts:10]: this text was used to click on a button lmao

**Review by davidfluck-hpe — APPROVED** [review]
Makes sense to me!

**Review by loksonarius — APPROVED** [review]
I mean, I dunno

**Comment by JComins000** [comment]
> I mean, I dunno

@loksonarius btw thing to look for would be "does it look like I'm replacing these things one-to-one" and "do the tests pass". thanks for looking!

**Pull request merged**